src/DssSpell.t.base.sol

```diff
@@ -260,17 +260,27 @@ contract DssSpellTestBase is Config, DssTest {
     }
 
     function _concat(string memory a, bytes32 b) internal pure returns (string memory) {
-        return string.concat(a, _bytes32ToStr(b));
+        return string.concat(a, _bytes32ToString(b));
     }
 
-    function _bytes32ToStr(bytes32 _bytes32) internal pure returns (string memory) {
-        bytes memory bytesArray = new bytes(32);
-        for (uint256 i; i < 32; i++) {
+    function _bytes32ToString(bytes32 _bytes32) internal pure returns (string memory) {
```

Comment:
The previous implementation would lead to trailing `\0` characters, which would show on Foundry logs like `MCD_PSM_USDC_A\0\0\0\0\0\0\0\0\0\0\0\0`.

The new one is more correct in a sense that it will get the length of the string first and stop at the appropriate place.
It is more expensive in terms of gas, but since this is used for testing purposes only, it is fine.